module/applications/components/inventory.mjs

```diff
@@ -202,6 +203,16 @@ export default class InventoryElement extends HTMLElement {
         condition: () => item.isOwner,
         callback: li => this._onAction(li[0], "delete")
       },
+      {
+        name: "DND5E.Scroll.CreateScroll",
+        icon: '<i class="fa-solid fa-scroll"></i>',
+        callback: async li => {
+          const data = await Item5e.createScrollFromSpell(item);
+          Item5e.createDocuments([data.toObject()], { parent: this.actor });
```

Comment:
We can pass the `Item5e` instance into `create`, it should have `toObject` called automatically.
```suggestion
          Item5e.create(data, { parent: this.actor });
```